### ISSUE TITLE
Mhp - The" Jobs" button , no of jobs  on the scraper is not the same as company's website

### ISSUE THREAD

**Issue by Geo843** [peviitor-ro/Scrapers_Matei#17]
ENVIRONMENT: production
URL: https://scrapers.peviitor.ro
Browser: Chrome
Device: Laptop
OS: Windows 10

STEPS TO REPRODUCE:

1. Open the URL in the browser
2. Search for "Mhp " Company
3. Check the "Jobs" location

EXPECTED RESULTS:

The "Jobs" location indicate the same no of jobs as company 's website.

ACTUAL RESULTS:
The" Jobs" button , no of jobs  on the scraper is not the same as company's website.

ATTACHMENTS:

![image](https://github.com/peviitor-ro/Scrapers_Matei/assets/71781611/1a5981c6-168a-4306-8beb-ac916aeb298d)

![image](https://github.com/peviitor-ro/Scrapers_Matei/assets/71781611/5b78f937-bff6-480e-855a-558c1f58d13d)

![image](https://github.com/peviitor-ro/Scrapers_Matei/assets/71781611/55fd8ee1-2db3-48db-b98a-83e732fbb7af)

**Comment by MateiBabaua** [comment]
Yes, because all 26 jobs are available in 2 locations, and I've duplicated 1 job across both locations
Ex.: 
![image](https://github.com/peviitor-ro/Scrapers_Matei/assets/146104320/1000315d-64d7-48e1-87f0-d4611efced74)
![image](https://github.com/peviitor-ro/Scrapers_Matei/assets/146104320/864bc659-6b81-48a1-b17f-1ccf7ec7c2fd)
